interface/main/tabs/menu/menus/standard.json

```diff
@@ -1756,36 +1864,28 @@
             "url": "/interface/reports/inventory_list.php",
             "target": "rep",
             "children": [],
-            "requirement": 0,
-            "acl_req": [
-              "admin",
-              "drugs"
-            ]
+            "requirement": 0
           },
           {
             "label": "Activity",
             "url": "/interface/reports/inventory_activity.php",
             "target": "rep",
             "children": [],
-            "requirement": 0,
-            "acl_req": [
-              "admin",
-              "drugs"
-            ]
+            "requirement": 0
           },
           {
             "label": "Transactions",
             "url": "/interface/reports/inventory_transactions.php",
             "target": "rep",
             "children": [],
-            "requirement": 0,
-            "acl_req": [
-              "admin",
-              "drugs"
-            ]
+            "requirement": 0
           }
         ],
         "requirement": 0,
+        "acl_req": [
+          ["admin", "drugs"],
+          ["inventory", "reporting"]
+        ],
         "global_req": "inhouse_pharmacy"
       },
```

Comment:
should also copy this file to:
sites/default/documents/custom_menus/Custom.json
(this is basically a duplicate of standard.json)

Reply:
OK thanks.